.github/workflows/prettier.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: |
           git fetch origin main
           # Check for changed JavaScript files.
-          CHANGED_FILES=$(git diff --name-only origin/main...HEAD | grep -E '\.js$|\.jsx$|\.ts$|\.tsx$' || echo "")
+          CHANGED_FILES=$(git diff --name-only --diff-filter=AM origin/main...HEAD | grep -E '\.js$|\.jsx$|\.ts$|\.tsx$' || echo "")
```

Comment:
@jazzsequence @rachelwhitton  here's an interesting thing, this workflow failed because it the diff included deleted files (and prettier can't run against a deleted file). So this filter displays only added or modified files.